### PR TITLE
Update Contributing

### PR DESCRIPTION
**Pull request by rcarteraz**
Add images for fiscal sponsors 

## Screenshots
Screenshot 2024-04-03 at 7 55 02 PM
Screenshot 2024-04-03 at 7 54 53 PM
Screenshot 2024-04-03 at 7 54 46 PM
Screenshot 2024-04-03 at 7 54 34 PM

### CODE REVIEW

**Comment by tekstrand** [comment]
Love the direction. It's going to be easier for me to push code to this branch than leave all the feedback here. We should use tailwind for all the styles here

File: tailwind.config.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import typography from "@tailwindcss/typography";
 import type { Config } from "tailwindcss";
 
 export default {
-  content: ["./src/**/*.{js,jsx,ts,tsx}"],
-  darkMode: "class",
+  content: ["./src/**/*.{js,jsx,ts,tsx,md,mdx}", "./docs/**/*.{md,mdx}"],
```
Comment (tekstrand) [inline tailwind.config.ts:5]: This change allows us to use css classes in MDX files without adding them to an exception or other `js` file. Any classes not currently used in js, jsx, ts, or tsx files will be removed from tailwind styles. Now the preprocessor looks for classes in md and mdx files and includes those in the final build

**Comment by tekstrand** [comment]
♻️ updated to use tailwind classes. It's working for dark/light mode and mobile viewports. Screenshots added to the PR description

File: docs/about/contributing.mdx [hunk 2]
```diff
@@ -78,10 +80,49 @@ If direct development contribution is not feasible for you, financial support is
...
 
-- **Open Collective & Open Source Collective**: Our collaboration with these organizations offers a robust fiscal management framework and banking solutions, supporting our project alongside over three thousand other open source initiatives. Open Collective's transparent framework lets everyone see our finances, including income, expenditures, and contributions on [Meshtastic's Open Collective page](https://opencollective.com/meshtastic). This transparency and support affirm that we're in excellent company.
-
-- **DigitalOcean**: Champions of open source, DigitalOcean supports Meshtastic through credits, which significantly aid our development, infrastructure, and testing efforts. This contribution is part of their commitment to fostering innovation and collaboration within the open source ecosystem.
-
-![This project is supported by DigitalOcean](https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/PoweredByDO/DO_Powered_by_Badge_blue.svg "This project is supported by DigitalOcean")
+  <div className="flex flex-wrap items-start">
+    <div className="w-full sm:w-3/12 p-3">
```
Comment (tekstrand) [inline docs/about/contributing.mdx:84]: `w-full` gives 100% width

`sm:w-3/12` means 25% width at screen sizes down to small (768px) in our case - so tablet and up will get two columns while mobile gets 1 stacked

`p-3` is a padding class

File: docs/about/contributing.mdx [hunk 3]
```diff
@@ -78,10 +80,49 @@ If direct development contribution is not feasible for you, financial support is
...
+      <ThemedImage
+        alt="Open Collective"
+        className="max-h-8"
```
Comment (tekstrand) [inline docs/about/contributing.mdx:87]: We set a maximum height here because if we don't the images are huge on mobile

**Review by caveman99 — APPROVED** [review]

**Pull request merged**